lib/awesome_bot/cli.rb

```diff
@@ -20,14 +20,27 @@ def make_option(o)
       "--#{o}"
     end
 
+    def divide(argv)
```

Comment:
@dkhamsing Here is the code to divide files and params but I know that it is not code.